test/compress/numbers.js

```diff
@@ -917,3 +917,17 @@ issue_3547_4: {
         "number 0",
     ]
 }
+
+unsafe_math_rounding: {
+    options = {
+        evaluate: true,
+        unsafe_math: true,
+    }
+    input: {
+        console.log(4 / -3 + 1 === 1 / -3);
```

Comment:
I don't think this input exercises unsafe_math.
```
$ echo 'console.log(4 / -3 + 1 === 1 / -3);' | bin/uglifyjs -c
console.log(!1);
```
The test case in https://github.com/mishoo/UglifyJS2/pull/3587#issuecomment-554604560 should be used.

Reply:
I'm mistaken - without this PR:
```
$ echo 'console.log(4 / -3 + 1 === 1 / -3);' | bin/uglifyjs -c unsafe_math
console.log(!0);
```
Please ignore.

Reply:
It's just a simplified version of https://github.com/mishoo/UglifyJS2/pull/3587#issuecomment-554604560 😉